dist/chart/values.yaml

```diff
@@ -177,7 +196,7 @@ metadata:
     tolerations: []
     affinity: {}
     redis:
-      host: aibrix-redis-master
+      host: "" # aibrix-redis-master
```

Comment:
there're test errors seems related to redis, please help take a look

Reply:
I think the issue is with this: https://github.com/vllm-project/aibrix/blob/c3e63f9b517ad18e38edeef03da327f979250b3d/pkg/cert/cert.go#L29

The rotator.go file is expecting a hard-coded name for the certificate, and the helm release installed was named: `aibrix-r8crnksvwc`, which is actually good because users should be able to choose their own helm release name.

I'll push a commit to set the webhook-server-cert name back to a hardcoded value to see if that fixes the test, but I think the code should be more flexible to reference the secret somehow, possibly using label selectors and namespace? Not sure.